src/libraries/System.Runtime/tests/System/DoubleTests.cs

```diff
@@ -272,6 +272,14 @@ public static IEnumerable<object[]> Parse_Valid_TestData()
             yield return new object[] { (-567.89).ToString(), defaultStyle, null, -567.89 };
             yield return new object[] { "1E23", defaultStyle, null, 1E23 };
 
+            yield return new object[] { ".234", defaultStyle, null, 0.234 };
+            yield return new object[] { "9813.", defaultStyle, null, 9813.0 };
+
+            string inputForDoubleWithDotAtMaxDigit = "6" + new string('1', 766) + ".";
+            yield return new object[] { inputForDoubleWithDotAtMaxDigit, defaultStyle, null, Double.Parse(inputForDoubleWithDotAtMaxDigit) };
```

Comment:
This is verifying that it doesn't throw, but should it actually verify the value that Double.Parse is returning?  Same for the other ones.